src/components/pages/ConnectPage.tsx

```diff
@@ -0,0 +1,312 @@
+  };
+
+  const openExternalLink = (url: string) => () => {
+    void open(url);
+  };
+
+  useEffect(() => {
+    dispatch(requestDisconnectFromAllDevices());
+    dispatch(requestAutoConnectPort());
+    requestPorts();
+  }, []);
+
+  // Wait to allow user to recognize serial connection succeeded
+  useEffect(() => {
+    if (
+      activePortState.status !== "SUCCESSFUL" &&
+      activeSocketState.status !== "SUCCESSFUL"
+    )
+      return;
+
+    const delayHandle = setTimeout(() => {
+      setScreenActive(false);
+    }, 600);
+
+    return () => {
+      clearTimeout(delayHandle);
+    };
+  }, [activePortState, activeSocketState]);
+
+  // Move to main page upon successful port connection (need to trigger when port is succesfully connected)
+  useEffect(() => {
+    if (isScreenActive) return;
+
+    const unmountHandle = setTimeout(() => {
+      unmountSelf();
+    }, 900);
+
+    return () => {
+      clearTimeout(unmountHandle);
+    };
+  }, [isScreenActive, unmountSelf]);
+
+  return (
+    <div
+      className="landing-screen-opacity-transition absolute flex flex-row h-screen w-screen z-40 bg-white"
+      style={{ opacity: isScreenActive ? 1 : 0 }}
+    >
+      <div className="flex flex-col flex-1 py-24 overflow-auto no-gutter">
+        <div className="flex justify-center">
+          <div className="h-1/8">
+            <img
+              className="w-11/12 h-11/12 text-gray-800"
+              src={Meshtastic_Logo}
+            ></img>
+          </div>
+        </div>
+
+        <div className="flex justify-center mt-10">
+          <h1 className="text-4xl font-semibold leading-10 text-gray-800">
+            Connect a radio
+          </h1>
+        </div>
+
+        <div className="flex justify-center mt-5">
+          <h2 className="text-base leading-6 font-normal text-gray-500 pl-40 pr-40 text-center">
+            Connect a supported Meshtastic radio to your computer via USB
+            Serial. For more detailed instructions,{" "}
+            <button
+              type="button"
+              onClick={() =>
+                void open("https://meshtastic.org/docs/introduction")
+              }
+              className="hover:cursor-pointer hover:text-gray-600 underline"
+            >
+              click here
+            </button>
+            .
+          </h2>
+        </div>
+
+        <div className="flex justify-center mt-5">
+          <Tabs.Root className="w-3/5" defaultValue={ConnectionType.SERIAL}>
+            <Tabs.List
+              className="flex flex-row"
+              aria-label="Choose a connection type"
+            >
+              <ConnectTab
+                label="Serial"
+                tooltip="Connect to your radio via a USB Serial cable"
+                value={ConnectionType.SERIAL}
+              />
+              <ConnectTab
+                label="TCP"
+                tooltip="Connect to your radio via ethernet or WiFi"
```

Comment:
Either it should be

```suggestion
                tooltip="Connect to your radio via a USB Serial cable"
                value={ConnectionType.SERIAL}
              />
              <ConnectTab
                label="TCP"
                tooltip="Connect to your radio via Ethernet or WiFi"
```

or "serial" should be lower case, too? You're the English pro :)

Reply:
Ah fair, good catch! I'll uppercase it

Reply:
Good catch, I'll update!